recipe/gen_patch_json.py

```diff
@@ -808,8 +808,25 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         if i >= 0:
             deps[i] = "cudatoolkit >=11.2,<12.0a0"
 
-        if "libiconv >=1.15,<1.16.0a0" in deps:
-            _pin_looser(fn, record, "libiconv", upper_bound="1.17.0")
+        if record.get('timestamp', 0) < 1662860143000:
```

Comment:
can you share how you generated this timestamp?

Reply:
python -c "import time; print(f'{time.time():.0f}000')"